packages/commonwealth/server/routes/getSubscribedChains.ts

```diff
@@ -14,14 +12,6 @@ export const getSubscribedChains = async (
   res: Response,
   next: NextFunction
 ) => {
-  if (!req.body.secret) {
-    return next(new AppError(Errors.NeedSecret));
-  }
-
-  if (req.body.secret != CHAIN_EVENT_SERVICE_SECRET) {
```

Comment:
Why reason why we removed this?

Reply:
Copied from response to Forest:
Because it's needed for querying from the client and it's not a dangerous operation.
We expose the chains anyways in the status route this is just a more specific version of that.
Originally it was meant solely for chain events to query but now it's useful on the client for determining possible chain subscriptions.